jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/security/FlowuiVaadinWebSecurity.java

```diff
@@ -170,4 +180,14 @@ protected void configure(WebSecurity web) throws Exception {
         super.configure(web);
         web.ignoring().requestMatchers(new AntPathRequestMatcher("/VAADIN/push/**"));
     }
+
+    protected RequestCache getDelegateRequestCache() {
+        HttpSessionRequestCache cache = new HttpSessionRequestCache();
+        cache.setRequestMatcher(createJmixViewPathRequestMatcher(viewRegistry));
+        return cache;
+    }
+
+    protected JmixViewPathRequestMatcher createJmixViewPathRequestMatcher(ViewRegistry viewRegistry) {
```

Comment:
Let's name method  `createViewPathRequestMatcher` and make return type `RequestMatcher`.